tests/integration/testlib/servicelib.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+from contextlib import contextmanager
+import time
+
+from . import cmdlib
+
+
+@contextmanager
+def disable_service(service):
+    cmdlib.exec_cmd(("systemctl", "stop", service))
+    try:
+        yield
+    finally:
+        cmdlib.exec_cmd(("systemctl", "start", service))
+        time.sleep(1)
```

Comment:
CI system might be slow. Please use a retry on `systemctl status openvswitch` return 0 with a timeout protected.

Reply:
I have some code at PR #967 :

```python

def retry_till_true_or_timeout(timeout, func, *args, **kwargs):
    ret = func(*args, **kwargs)
    while timeout > 0:
        if ret:
            break
        time.sleep(1)
        timeout -= 1
        ret = func(*args, **kwargs)
    return ret
```

Reply:
You are right. I fixed it. Thanks for the code example.